scripts/travis-emscripten/base64.js

```diff
@@ -0,0 +1,48 @@
+/*\
```

Comment:
License?

Reply:
Yeah, good question... Judging from https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses about "Code samples and snippets", *probably* ``public domain (CC0) No licensing notice is necessary`` applies...